src/components/DataTable/DataTable.component.tsx

```diff
@@ -112,11 +115,19 @@ const DataTableRow = <Data extends BaseRowData = BaseRowData>({
 const LoadingStateRow = <Data extends BaseRowData = BaseRowData>({
   columns,
 }: LoadingStateRowProps<Data>) => (
-  <TableRow>
+  <TableRow data-testid="loading-state-row">
     {columns.map((_, cellIndex) => (
       <TableCell key={cellIndex}>
         <Skeleton />
       </TableCell>
     ))}
   </TableRow>
 );
+
+const TableEmpty: React.FC = () => (
+  <TableRow>
+    <Box component="td" py={3} px={1.25} color="rgba(255,255,255,0.5)">
+      No stakes yet.
```

Comment:
This text should be passed as a prop. There will be different text in case of current stakes and current vests tables